languages/react-native/rusaint-react-native.podspec

```
@@ -0,0 +1,44 @@
+folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
+
+Pod::Spec.new do |s|
+  s.name         = "rusaint-react-native"
+  s.version      = package["version"]
+  s.summary      = package["description"]
+  s.homepage     = package["homepage"]
+  s.license      = package["license"]
+  s.authors      = package["author"]
+
+  s.platforms    = { :ios => min_ios_version_supported }
+  s.source       = { :git => "https://github.com/EATSTEAK/rusaint.git", :tag => "#{s.version}" }
```

RuboCop (1.75.5)

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)